src/PantheonTerminalWindow.vala

```diff
@@ -139,6 +137,11 @@ namespace PantheonTerminal {
             action_accelerators[ACTION_ZOOM_IN_FONT] = "<Control>KP_Add";
             action_accelerators[ACTION_ZOOM_OUT_FONT] = "<Control>minus";
             action_accelerators[ACTION_ZOOM_OUT_FONT] = "<Control>KP_Subtract";
+            action_accelerators[ACTION_COPY] = "<Control><Shift>c";
+            action_accelerators[ACTION_PASTE] = "Control><Shift>v";
+            action_accelerators[ACTION_SEARCH] = "<Control><Shift>f";
+            action_accelerators[ACTION_SELECT_ALL] = "<<Control><Shift>a";
```

Comment:
I think the line 143 should be `action_accelerators[ACTION_SELECT_ALL] = "<Control><Shift>a";`.
Because after this update was installed to my environment, pressing `Shift + a` selects all in the terminal and pressing `Ctrl + Shift + a` do nothing.

Reply:
I'm sorry, it fixed just now #257